tests/unit/storage/utils.js

```diff
@@ -0,0 +1,24 @@
+const assert = require('assert');
+const { serializeCircJSON } = require('../../../lib/storage/utils');
+
+const circularJSON = {
+    key1: 'foo',
+    key2: 1,
+    key3: { key4: 5 },
+};
+circularJSON.key5 = circularJSON;
+
+describe('SerializeCircJSON', () => {
+    it('should stringify circular JSON', () => {
```

Comment:
Can you add a test stringifying a regular json and a test for invalid json?

Reply:
Added